uint8 RESUMING=0
uint8 ACTIVE=1
uint8 PAUSED=2

Pose last_pose
Point2D[] absolute_points
float32[] distances_from_origin
int32 total_points
float32 total_distance
int32 last_point_touched
float32 current_edge_travelled_distance
float32 percentage_completed
uint8 drawing_status
